modules/alloy/src/main/ts/ephox/alloy/ui/schema/DropdownSchema.ts

```diff
@@ -35,7 +35,13 @@ const schema = Fun.constant([
 const parts: () => PartType.PartTypeAdt[] = Fun.constant([
   PartType.external<DropdownDetail, DropdownSpec>({
     schema: [
-      Fields.tieredMenuMarkers()
+      Fields.tieredMenuMarkers(),
+      // INVESTIGATE: I believe defining a defaulted field isn't necessary
```

Comment:
Does this need a ticket to investigate?

Reply:
Nope. I can just remove the `INVESTIGATE` part. I had a look later.

Reply:
- [ ] rewrite comment.

Reply:
Updated: rewrote comment.